.github/workflows/cron-correctness.yml

```diff
@@ -8,42 +8,4 @@ on:
 
 jobs:
   bindingtester:
-    name: Binding Tester
-    runs-on: ubuntu-latest
-    if: github.repository == 'foundationdb-rs/foundationdb-rs'
-
-    strategy:
-      matrix:
-        parallelism: ["1", "2", "3"]
-    env:
-      CARGO_INCREMENTAL: 0
-      RUST_BACKTRACE: 1
-
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v1
-        with:
-          submodules: recursive
-
-      - name: Setup FoundationDB
-        uses: foundationdb-rs/foundationdb-actions-install@v2.0.0
-        with:
-          version: "6.3.23"
-
-      - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          profile: minimal
-
-      - name: Build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: -p bindingtester
-
-      - name: Setup bindingtester
-        run: scripts/setup_bindingtester.sh target/debug/bindingtester
-
-      - name: Run bindingtester
-        run: scripts/run_bindingtester.sh 300
+    uses: foundationdb-rs/foundationdb-rs/.github/workflows/correctness.yml@main 300
```

Comment:
I think you need to use the [`with`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepswith) syntax here:

```yml
jobs:
  bindingtester:
    name: Run bindingtester
    uses: foundationdb-rs/foundationdb-rs/.github/workflows/correctness.yml@main 1000 
    with:
      iterations: 300
```